src/shader_recompiler/ir/passes/resource_tracking_pass.cpp

```diff
@@ -711,7 +738,12 @@ void PatchImageInstruction(IR::Block& block, IR::Inst& inst, Info& info, Descrip
 
     if (inst_info.has_lod) {
         ASSERT(inst.GetOpcode() == IR::Opcode::ImageFetch);
-        inst.SetArg(3, arg);
+        inst.SetArg(3, arg1);
+        if (image.GetType() == AmdGpu::ImageType::Color2DMsaa) {
```

Comment:
MS textures don't have mips, so I would prefer to have an assertion for this case instead